constitution.tex

```diff
@@ -770,6 +770,20 @@
 \item Candidates must be Active Members
 \end{enumerate}
 
+\asubsection{Waiving of Requirements}
+\begin{enumerate}
+  \item The following requirements may be waived as decided by a Two-Thirds Immediate Vote as described in \ref{Two-Thirds} and \ref{Immediate Vote}:
+    \begin{enumerate}
+      \item Candidates must reside on floor during the term of office
+      \item Candidates must have at least one full semester of House membership as an Active Member
+      \item Social and Research and Development are the only voting Executive Board positions that allow for Dual Directorship.
+    \end{enumerate}
+  \item This process operates for one requirement at a time
+  \item This process operates for one voting executive board position or the Chairperson at a time
+  \item This process operates on the current election only
+  \item Any requirements waived using this process are waived for all candidates for the given position for the current election process
```

Comment:
What about rephrasing this like follows (plus it lets you get rid of the double enumerate)
```suggestion
Each vote to waive a qualification may only apply to a single qualification and a single Selection.
The chair of the vote shall state what qualification waiver is being voted upon, what Selection the waiver will apply to, and any nominated candidates not qualified under \ref{Qualifications} who would become qualified if if the waiver passes.
```

I'd also move the setting of quorum requirements and voting methods to the end.

Reply:
Honestly I don't see a clean way to move quorum / voting methods down...

Reply:
Maybe something along the lines of the following? I did some rearranging and rewording.

```tex
\asubsection{Waiving of Qualifications}
House may choose to waive the following subset of the Qualifications for Executive Board positions (defined under \ref{Qualifications}).
\begin{enumerate}
  \item Candidates must reside on floor during the term of office.
  \item Candidates must have at least one full semester of House membership as an Active Member.
  \item Social and Research and Development are the only voting Executive Board positions that allow for Dual Directorship.
\end{enumerate}
When a waiver is proposed, the Chair of the Vote shall state what qualification waiver is being voted upon, what Selection the waiver will apply to, and any nominated candidates not qualified under \ref{Qualifications} who would become qualified if the waiver passes.
A Two-Thirds Immediate Vote, as defined in \ref{Two-Thirds} and \ref{Immediate Vote}, is then taken to determine whether the proposed waiver shall take effect.
Each vote to waive a Qualification must only apply to a single Qualification in a single Selection.
As a waiver applies to the Selection, and not to individual candidates, it applies to all candidates involved in the Selection, regardless of their other qualifications or prior nominations.
```